fuse/cross_agency.py

```diff
@@ -54,20 +54,18 @@ def assign_max_col(events: pd.DataFrame, per: pd.DataFrame, col: str):
 
 def read_constraints():
     # TODO: replace this line with the real constraints data
-    constraints = pd.DataFrame([], columns=["type", "uids"])
+    constraints = pd.read_csv(deba.data("clean/post_officer_history.csv"))
```

Comment:
Are we really going to retire the contraints table? I think we still might want to manually edit the result from time to time. It's better to introduce POST history as an additional input.

Reply:
Ok.